Gemfile

```diff
@@ -12,6 +12,7 @@ gem 'database_cleaner', '~> 1.3.0', require: false
 group :development do
   gem 'guard'
   gem 'guard-rspec', require: false
+  gem 'pry-rescue'
```

Comment:
I didn't intend to leave it there. I can remove it if people prefer, but I've found `pry-rescue` useful in debugging.